index.js

```diff
@@ -1 +1,101 @@
-console.log('Hello, Node!')
+const fs = require("fs");
+const path = require("path");
+const petsFile = path.join(__dirname, "data", "pets.json");
+const contents = fs.readFileSync(petsFile, "utf-8");
+const pets = JSON.parse(contents);
+
+// Utility pet checker
+const runPetSearch = name => {
+  let isPetRecord = pets.some(record => record.name === name);
+
+  return isPetRecord;
+};
+
+// Read contents
+const read = () => pets;
```

Comment:
You are reading the contents of the JSON file only once at the top of index on line 5, this function will only point to the original snapshot of the file. Instead you should be reparsing the file within this read function every time it is called.